deps/dicer/lib/HeaderParser.js

```diff
@@ -79,14 +79,19 @@ HeaderParser.prototype._parseHeader = function () {
         continue
       }
     }
+
+    const posColon = lines[i].indexOf(':')
+    if (
+      posColon === -1 ||
+      posColon === 0
+    ) {
+      return
```

Comment:
shouldn't we throw an error instead? is this even valid payload?

Reply:
would be helpful to at least log an error then silent ignoring may be confusing and hard to debug

Reply:
We have no logging functionality in busboy :/
Also before there was no logging of the invalid header.

Reply:
we should add some. not in this pr, though

Reply:
thanks for the mercy :)